package.yml

```
@@ -12,42 +15,41 @@
      default: "y"
      private: True
   tasks:
-    - name: make sure packages are installed
-      apt:
-        update_cache: yes
-        force_apt_get: yes
-        name:
-          - 'git-core'
-          - 'python-dev'
-          - 'postgresql'
-          - 'libpq-dev'
-          - 'libxml2-dev'
-          - 'libxslt1-dev'
-          - 'build-essential'
-          - 'apache2'
-          - 'libapache2-mod-wsgi'
-          - 'rubygems-integration'
-          - 'ruby-dev'
-
-
-    - name: stop apache before installing nginx (xenial)
+    - name: set regex for CKAN version
+      set_fact:
+        re: '\D+\-\D+(?P<capture>\d+\.\d+)'
+    - set_fact:
+        ckanver: "{{ version | regex_search(re, '\\g<capture>') }}"
+        register: ckanver
+    - debug:
+        var: ckanver[0]
+    
+    # Install Ubuntu apt Packages
+    - name: Install apt Packages 
+      include_tasks: "py2/package.yml"
+      when: pythonversion == "py2"
+    - name: Install Packages 
+      include_tasks: "py3/package.yml"
+      when: pythonversion == "py3"
```

```suggestion
    - name: Install apt Packages 
      include_tasks: "{{ pythonversion }}/package.yml"
```

This will save us some lines and make the file more readable. You can do the same for the version dependant includes (eg 2.7/after_web.sh, 2.8/build.yml, etc)